src/components/select/bl-select.ts

```diff
@@ -241,58 +330,74 @@ export default class BlSelect extends LitElement {
       'select-wrapper': true,
       'select-open': this.opened,
       'selected': this._selectedOptions.length > 0,
-      'invalid': this._isInvalid,
+      'invalid': !this.validity.valid,
+      'dirty': this.dirty
     })}
-      tabindex="-1"
+      @keydown=${this.handleKeydown}
     >
-      ${label} ${this.inputTemplate()} ${this.menuTemplate()} ${helpMessage} ${invalidMessage}
+      ${label}
+      ${this.inputTemplate()}
+      <div class="popover" tabindex="${ifDefined(this._isPopoverOpen ? undefined : '-1')}" @bl-select-option=${this._handleSelectOptionEvent}>
+        <slot></slot>
+      </div>
+      <div class="hint">${invalidMessage} ${helpMessage}</div>
     </div> `;
   }
 
-  private get _showPlaceHolder() {
-    if (this.label && !this.labelFixed) {
-      return !this._selectedOptions.length && this._isPopoverOpen;
+  private focusedOptionIndex = -1;
+
+  private handleKeydown(event: KeyboardEvent) {
+    if (this.focusedOptionIndex === -1 && event.code === 'Enter' || event.code === 'Space') {
```

Comment:
How about having `ArrowDown` as well to toggle the popup specially for this component?

Reply:
Hmm, good point. I'll check the native select if it has a behavior like this.

Reply:
`ArrowDown` is also opening select from now on as in native select input.